### ISSUE TITLE
403 Following Harbinger League Rollover

### ISSUE THREAD

**Issue by mattmarich** [Stickymaddness/Procurement#766]
System.Net.WebException: The remote server returned an error: (403) Forbidden.
   at System.Net.HttpWebRequest.GetResponse()
   at POEApi.Transport.HttpTransport.GetStash(Int32 index, String league, Boolean refresh)
   at POEApi.Transport.CachedTransport.GetStash(Int32 index, String league, Boolean refresh)
   at POEApi.Model.POEModel.GetStash(Int32 index, String league, Boolean forceRefresh)
   at POEApi.Model.POEModel.GetStash(String league)
   at Procurement.ViewModel.LoginWindowViewModel.loadStash(Character character)
   at Procurement.ViewModel.LoginWindowViewModel.<Login>b__1()
   at System.Threading.Tasks.Task.Execute()

**Comment by mattmarich** [comment]
chokes when it starts to download Standard Stash Tab 1

**Comment by Stickymaddness** [comment]
Weird, are you still getting this?

**Comment by mattmarich** [comment]
I actually don't make it to that point now. When loading my first character I'm presented with this.

```Parameter name: source
   at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
   at POEApi.Model.Gear..ctor(Item item)
   at POEApi.Model.ItemFactory.Get(Item item)ItemFactory unable to instanciate type : Crystal BeltSystem.Exception: ItemFactory unable to instanciate type : Crystal Belt
   at POEApi.Model.ItemFactory.Get(Item item)
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at POEApi.Model.POEModel.GetInventory(String characterName)
   at Procurement.ViewModel.LoginWindowViewModel.loadCharacterInventory(Character character)
   at Procurement.ViewModel.LoginWindowViewModel.<Login>b__1()
   at System.Threading.Tasks.Task.Execute()```

**Comment by mattmarich** [comment]
nevermind, must have fixed this in 1.18.0
